lib/pbench/test/unit/server/test_controllers_list.py

```diff
@@ -171,14 +174,54 @@ def test_query(self, client, server_config, query_helper, user_ok):
         assert res_json[1]["last_modified_value"] == 1.6
         assert res_json[1]["last_modified_string"] == "2020-09-26T20:19:15.810Z"
 
+    def test_empty_query(self, client, server_config, query_helper, user_ok):
+        """
+        Check proper handling of a query resulting in no Elasticsearch matches.
+        """
+        json = {
+            "user": "drb",
+            "start": "2020-08",
+            "end": "2020-10",
+        }
+        response_payload = {
+            "took": 1,
+            "timed_out": False,
+            "_shards": {"total": 1, "successful": 1, "skipped": 0, "failed": 0},
+            "hits": {
+                "total": {"value": 0, "relation": "eq"},
+                "max_score": None,
+                "hits": [],
+            },
+        }
+
+        index = self.build_index(server_config, ("2020-08", "2020-09", "2020-10"))
+        response = query_helper(
+            json, index, HTTPStatus.OK, server_config, json=response_payload
+        )
+        res_json = response.json
+        assert isinstance(res_json, list)
+        assert len(res_json) == 0
```

Comment:
Why not just
```
        assert res_json == [], f"Received {res_json!r} when expecting an empty list."
```

Reply:
Because ... because ... because ... then everyone would know what it meant??